PACKAGES.md

```diff
@@ -0,0 +1,27 @@
+### Most Core Packages & Examples
```

Comment:
Do you think we should try to (sub)group/categorize these by their purpose?  For example, `prelude` is a very different kind of thing than `hold`.  Maybe we should group "tools for most.js contributors" like `prelude` and `package-starter`, and "most.js stream add-ons" or some such?

Does that seem like it'd be helpful?